src/adapters/supabase/types/database.ts

```diff
@@ -0,0 +1,367 @@
+export type Json =
```

Comment:
it seems supabase generated types violate our eslint rules

Reply:
We can `.eslintignore` generated files from Supabase

Reply:
I think we should avoid committing these